sdk/eventgrid/Azure.Messaging.EventGrid/tests/EventGridClientUnitTests.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.Messaging.EventGrid.Tests
+{
+    public class EventGridClientUnitTests : ClientTestBase
+    {
+        public EventGridClientUnitTests(bool async) : base(async)
+        {
+        }
+
+        [Test]
+        public void MustSetKeyCredential()
```

Comment:
nit: typically we name tests that are asserting an exception is thrown something like BuildSharedAccessSignatureThrowsWhenCalledFromSasClient